guidelines/relative-luminance.html

```diff
@@ -242,7 +242,7 @@ <h1>MathML version of the relative luminance definition</h1>
               </msub>
               <mo>,</mo>
               <mrow>
-                <mspace width="1ex"/>
+                <mspace width="1ex">
```

Comment:
Looks like this only ended up fixed for one and not the other?
```suggestion
                <mspace width="1ex"/>
```